hooks/useRoomActions.ts

```diff
@@ -103,9 +103,16 @@ export default function useRoomActions() {
 
   function onEndTurn() {
     runTransaction(roomRef, (room: Room) => {
+      var newStatus: RoomStatus;
+      if (room.settings.maxRounds == room.round && room.currentTeamIndex == room.teams.length - 1){
+        newStatus = "ended"
```

Comment:
This line seems to be used both here and in room atom. Let's extract it out to a well-named util function and use that instead.